leshan-server-core/src/main/java/org/eclipse/leshan/server/bootstrap/BootstrapSessionManager2.java

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Sierra Wireless and others.
+ * 
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * and Eclipse Distribution License v1.0 which accompany this distribution.
+ * 
+ * The Eclipse Public License is available at
+ *    http://www.eclipse.org/legal/epl-v20.html
+ * and the Eclipse Distribution License is available at
+ *    http://www.eclipse.org/org/documents/edl-v10.html.
+ * 
+ * Contributors:
+ *     Sierra Wireless - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.leshan.server.bootstrap;
+
+import org.eclipse.leshan.core.request.BootstrapRequest;
+import org.eclipse.leshan.core.request.Identity;
+
+/**
+ * A new version of {@link BootstrapSessionManager}.
+ * 
+ * @since 1.1
+ */
+public interface BootstrapSessionManager2 extends BootstrapSessionManager {
```

Comment:
Same question for this PR. Can't we update the existing interface and mark the old method as deprecated?

Reply:
Unfortunately, [default method was added since java8](https://www.oracle.com/java/technologies/javase/8-whats-new.html). And for now we support java7. (that was mainly an android constraint) 

Maybe for the 2.0 we should consider to move to java8.